packages/terra-graphs-docs/src/example-datasets/dataObjects/Gantt/barTypesDataset2.js

```diff
@@ -0,0 +1,28 @@
+import {
+  loadXAndYAxisLabelPopup,
+  loadTaskPopup,
+} from '@cerner/terra-graphs-docs/lib/example-datasets/popup';
+
+const barTypesDataset2 = {
+  key: 'track 1',
```

Comment:
nit: keep key and object name similar

Reply:
@sdadn in Gantt I guess track 1 sounds more meaningful than barTypesDataset.

Reply:
at the very least, keep the numbers the same

Reply:
Actually, here track starts from 0 so the number looks a bit different.